File: C/somavetor.c
#include <stdio.h>
int main()
{
    int e;
    double vet[e],soma;
    soma=0;
    printf("Quantos numeros voce vai digitar? ");
    scanf("%i",&e);
        for ( int i = 0; i <= e-1; i++)
        {
         printf("Digite um numero: ");
         scanf("%lf",&vet[i]);
        }
        printf("valores = ");
        for ( int i = 0; i <= e-1; i++)
        {
            soma=soma+vet[i];
            printf("%.1lf ",vet[i]);
        }
        printf("\nSOMA = %.2lf\n",soma);
        printf("MEDIA = %.2lf",soma/e);
    return 0;
}    

File: Phyton/old/somavetor.py
n=int(input("Quantos numeros serao digitados: "))
vet = [0 for x in range (n)]
for i in range(0,n):
    vet[i]=float(input("digite um numero: "))
print()
print("Valores= ",end="")
soma=0
for i in range(0,n):
    print(f"{vet[i]:.1f} ",end="")
    soma=soma+vet[i]
print()
print(f"soma = {soma:.2f}")
media=soma/n
print(f"media = {media}")